apps/web/src/app/dashboard/teacher/quiz/_components/Cards.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  CardFooter,
+  CardTitle
+} from "@/components/ui/card";
+import Link from "next/link";
+import { trpc } from "../../../../../utils/trpc/client";
+import { TrpcReactQueryOptions } from "../../../../../utils/trpc/lib";
+import { NewQuizButton } from "./NewQuizButton";
+
+interface Props {
+  initialData: TrpcReactQueryOptions["questionBank"]["list"]["initialData"];
+}
+
+export const QuestionBankCards = ({ initialData }: Props) => {
+  const { data: questionsBanks, isLoading } = trpc.questionBank.list.useQuery(
+    undefined,
+    {
+      initialData
+    }
+  );
+  console.log(questionsBanks);
```

Comment:
can remove

Reply:
Resolved in 
https://github.com/peiyuans-pals/toohakai/commit/19647d875eb27ab17a412917c24f814f0f436a1f